### DISCUSSION TITLE
"Beats-per-minute" standard ID3 tag is not available?

### DISCUSSION THREAD

**Discussion by mdkberry** [strawberrymusicplayer/strawberry#1447] [Ideas]
Given how important EDM is as a music genre, and how important BPM is to EDM as a music filter, how come we don't have the "beats-per-minute" tag available in the columns list of main window? Seems incredible this is not in there. I absolutely 100% love this software so far, but absolutely 100% need BPM on my music as a tag filter.

**Comment by jonaski** [comment]
BPM is added in newer versions.
